base_station/gui/src/components/EncoderCounts.vue

```diff
@@ -1,11 +1,14 @@
  <template>
-  <div class="encoders">
-    <div>joint a: {{Math.round((encoderCounts.joint_a + Number.EPSILON) * 10000) / 10000}}</div>
-    <div>joint b: {{Math.round((encoderCounts.joint_b + Number.EPSILON) * 10000) / 10000}}</div>
-    <div>joint c: {{Math.round((encoderCounts.joint_c + Number.EPSILON) * 10000) / 10000}}</div>
-    <div>joint d: {{Math.round((encoderCounts.joint_d + Number.EPSILON) * 10000) / 10000}}</div>
-    <div>joint e: {{Math.round((encoderCounts.joint_e + Number.EPSILON) * 10000) / 10000}}</div>
-    <div>joint f: {{Math.round((encoderCounts.joint_f + Number.EPSILON) * 10000) / 10000}}</div>
+  <div class="wrap">
+    <h3>Encoder Counts</h3>
+    <div class="encoders">
+      <div>joint a: {{Math.round((encoderCounts.joint_a + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint b: {{Math.round((encoderCounts.joint_b + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint c: {{Math.round((encoderCounts.joint_c + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint d: {{Math.round((encoderCounts.joint_d + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint e: {{Math.round((encoderCounts.joint_e + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint f: {{Math.round((encoderCounts.joint_f + Number.EPSILON) * 10000) / 10000}}</div>
```

Comment:
Can you fix this rounding? I think I implemented this rounding really quickly and didn't realize I could just use .toFixed(5)